pkg/config/config.go

```diff
@@ -118,3 +118,14 @@ func ParseConfig() error {
 	logrus.Debugf("params = %v ", Parameters)
 	return nil
 }
+
+// Copy will create a flat copy of GenericMap
+func (m GenericMap) Copy() map[string]interface{} {
+	result := map[string]interface{}{}
+
+	for k, v := range m {
+		result[k] = v
+	}
+
+	return result
+}
```

Comment:
I do a simple flat copy here for performances purpose but we can implement a deep copy if needed

Reply:
Should this function be here or in utils?

Reply:
@KalmanMeth @jpinsonneau I had the same thought .... maybe the right thing to do is open a new GenericMap.go file

Reply:
I can move this :+1: 

Does a new file `generic_map.go` in config folder is ok for both of you ? @KalmanMeth @eranra

Reply:
https://github.com/netobserv/flowlogs-pipeline/pull/226/commits/2648b82480b2f4a2515e3bee5055568afa7a47be#diff-2eff304122c33ebdc65982528ca9efc9fd8061e6c75fda0beffd1b3ce0cdc046